learn/advanced-concepts/03-DAG-Token.md

```diff
@@ -4,63 +4,43 @@ sidebar_label: DAG Token
 hide_table_of_contents: false
 ---
 
-The Hypergraph's native cryptocurrency is known as $DAG. The $DAG operates on an exclusive metagraph that facilitates seamless interaction between metagraphs and network nodes. In other words, $DAG binds network components together and is how the Hypergraph provides utility.
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
 
-## DAG L1 Network​
-DAG Layer 1 (L1) is a special case metagraph that validates DAG currency transactions for the network and bundles them into blocks which are then submitted to the global L0 network for final validation. This metagraph enables $DAG which is the native currency of the network
+##
 
-The DAG L1 network is a special case metagraph that validates transactions, runs them through parallel rounds of consensus, and processes them into blocks that are then submitted to the Global L0 network for inclusion in a Global Snapshot. DAG is the native currency of the network.
+DAG is the Hypergraph's native cryptocurrency. The [$DAG token](https://constellationnetwork.io/dag-token) operates on an exclusive metagraph that facilitates seamless interaction between metagraphs and network nodes. In other words, $DAG binds network components together and is how the Hypergraph provides utility.
 
-### What is $DAG ?
+The mission of the DAG token, is to promote Constellation's Network by incentivizing participation from all stakeholders, from Validator Nodes to metagraphs.
```

Comment:
The DAG L1 isn't a metagraph and it doesn't really facilitate interaction between metagraphs yet. I would keep the previous text that this replaced but just remove the out-there/extra stuff like "rave reviews", "sought after assets", etc. I would also change "$DAG works as an exclusive dApp" to "DAG operates on a special-case Layer 1 network that interacts directly with the Global L0"